cmd/list_values.go

```diff
@@ -0,0 +1,163 @@
+package cmd
+
+import (
+	"fmt"
+
+	log "github.com/charmbracelet/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	e "github.com/cloudposse/atmos/internal/exec"
+	"github.com/cloudposse/atmos/pkg/config"
+	l "github.com/cloudposse/atmos/pkg/list"
+	fl "github.com/cloudposse/atmos/pkg/list/flags"
+	f "github.com/cloudposse/atmos/pkg/list/format"
+	"github.com/cloudposse/atmos/pkg/schema"
+	u "github.com/cloudposse/atmos/pkg/utils"
+)
+
+var (
+	ErrGettingCommonFlags    = errors.New("error getting common flags")
+	ErrGettingAbstractFlag   = errors.New("error getting abstract flag")
+	ErrGettingVarsFlag       = errors.New("error getting vars flag")
+	ErrInitializingCLIConfig = errors.New("error initializing CLI config")
+	ErrDescribingStacks      = errors.New("error describing stacks")
+)
+
+// Error format strings.
+const (
+	ErrFmtWrapErr = "%w: %v" // Format for wrapping errors.
+)
+
+// listValuesCmd lists component values across stacks.
+var listValuesCmd = &cobra.Command{
+	Use:   "values [component]",
+	Short: "List component values across stacks",
+	Long:  "List values for a component across all stacks where it is used",
+	Example: "atmos list values vpc\n" +
+		"atmos list values vpc --abstract\n" +
+		"atmos list values vpc --query '.vars'\n" +
+		"atmos list values vpc --query '.vars.region'\n" +
+		"atmos list values vpc --format json\n" +
+		"atmos list values vpc --format yaml\n" +
+		"atmos list values vpc --format csv",
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			log.Error("invalid arguments. The command requires one argument 'component'")
+			return
+		}
+
+		// Check Atmos configuration
+		checkAtmosConfig()
+		output, err := listValues(cmd, args)
+		if err != nil {
+			log.Error(err.Error())
+			return
+		}
+
+		u.PrintMessage(output)
+	},
+}
+
+// listVarsCmd is an alias for 'list values --query .vars'.
+var listVarsCmd = &cobra.Command{
+	Use:   "vars [component]",
+	Short: "List component vars across stacks (alias for 'list values --query .vars')",
```

Comment:
We support markdown.

```suggestion
	Short: "List component vars across stacks (alias for `list values --query .vars`)",
```